rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -320,31 +319,6 @@ def test_update_snapshot_balances(task_manager):
         raise AssertionError(f'Update snapshot balances was not completed within {timeout} seconds') from e  # noqa: E501
 
 
-@pytest.mark.parametrize('globaldb_upgrades', [[]])
-@pytest.mark.parametrize('run_globaldb_migrations', [False])
-@pytest.mark.parametrize('custom_globaldb', ['v4_global_before_migration1.db'])
```

Comment:
Is `v4_global_before_migration1.db` used anywhere else? If not you should also remove that file.

Reply:
Yes, it is used in a few other tests too so we still need it